trite-osprey/src/App.css

```diff
@@ -70,17 +69,54 @@ ul li:hover{
 
 }
 
+.center_section{
+  display: flex;
+  justify-content: space-around;
+
+}
+.container_left{
+  display:flex;
+  justify-content: space-around;
+}
+.container_right{
+  display: flex;
+  justify-content: space-around;
+  align-items: flex-start;
+  flex-direction: column;
+}
 .popout {
+  display:flex;
+  flex-direction: column;
   z-index: 2;
   width: 500px;
   height: 500px;
-  border:6px solid black;
+  border:6px solid #222;
   background-color: #fff;
   position: absolute;
   top: 275px;
   left: 320px;
+  /*padding:1em;*/
 }
 
 .invisible {
   display: none;
 }
+
+button{
+  color: white;
+  background-color: #222;
+}
+.cartContainer{
+ display: flex;
+ flex-direction: row;
+ justify-content: flex-end;
+ padding:2em;
+}
+.price{
+  display: flex;
+  flex-direction: column;
+}
+.beercartlist{
+  display: flex;
+  flex-direction: row;
+}
```

Comment:
I see display flex on almost every class. Do all of these have children components? If any do not have children do not need flexboxs.